mibs/eltex/eltMes.mib

```diff
@@ -0,0 +1,152 @@
+--
+-- Eltex MES Ethernet Switches  MIBs
+--
+-- Copyright (c) 2015, Eltex Ltd
+-- All rights reserved.
+--
+
+ELTEX-MES DEFINITIONS ::= BEGIN

```

Comment:
You should rename all these mib files to what's written before `DEFINITIONS`, so for this file the name should be `ELTEX-MES` (Without .mib)

Reply:
ok @Jellyfrog 
i renamed MIB files